recipes/xapian-core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xapian-core" %}
+{% set version = "1.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://oligarchy.co.uk/xapian/{{ version }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 13f08a0b649c7afa804fa0e85678d693fd6069dd394c9b9e7d41973d74a3b5d3
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: |
+    ./configure \
+    CXXFLAGS=-I$PREFIX/include \
+    LDFLAGS=-L$PREFIX/lib \
```

Comment:
Change to `LDFLAGS=$LDFLAGS`

Reply:
compiler package already sets these variables

Reply:
I agree with @isuruf, there are probably not correct as-is

Reply:
That gives
```
configure: error: unrecognized option: `-Wl,--sort-common'
```

Reply:
I can try putting the `LDFLAGS` before the `./configure` command, that might prevent the script from interpreting additional unknown options. (In general, though, I seem to remember that it's recommended to put custom compiler flags as arguments to the script.)

Reply:
OK, prepending `LDFLAGS` works fine on my machine. I've updated the recipe accordingly.